testsuite/CMakeLists.txt

```diff
@@ -1,193 +1,2 @@
-python_test(FILE rescale.py MAX_NUM_PROC 2)
-python_test(FILE npt.py MAX_NUM_PROC 4)
-python_test(FILE elc_vs_mmm2d_neutral.py MAX_NUM_PROC 2)
-python_test(FILE elc_vs_mmm2d_nonneutral.py MAX_NUM_PROC 2)
-python_test(FILE accumulator.py MAX_NUM_PROC 4)
-python_test(FILE wang_landau_reaction_ensemble.py MAX_NUM_PROC 1)
-python_test(FILE array_properties.py MAX_NUM_PROC 4)
-python_test(FILE analyze_distribution.py MAX_NUM_PROC 1)
-python_test(FILE observable_profile.py MAX_NUM_PROC 4)
-python_test(FILE observable_profileLB.py MAX_NUM_PROC 1)
-python_test(FILE rotate_system.py MAX_NUM_PROC 4)
-python_test(FILE random_pairs.py MAX_NUM_PROC 4)
-python_test(FILE lb_electrohydrodynamics.py MAX_NUM_PROC 4)
-python_test(FILE cluster_analysis.py MAX_NUM_PROC 4)
-python_test(FILE pair_criteria.py MAX_NUM_PROC 4)
-python_test(FILE actor.py MAX_NUM_PROC 1)
-python_test(FILE drude.py MAX_NUM_PROC 2)
-python_test(FILE thermalized_bond.py MAX_NUM_PROC 4)
-python_test(FILE thole.py MAX_NUM_PROC 4)
-python_test(FILE lb_switch.py MAX_NUM_PROC 1)
-python_test(FILE lb_boundary_velocity.py MAX_NUM_PROC 1)
-python_test(FILE lb_thermo_virtual.py MAX_NUM_PROC 2)
-python_test(FILE lb_poiseuille.py MAX_NUM_PROC 4)
-python_test(FILE lb_interpolation.py MAX_NUM_PROC 4)
-python_test(FILE analyze_gyration_tensor.py MAX_NUM_PROC 1)
-python_test(FILE oif_volume_conservation.py MAX_NUM_PROC 2)
-python_test(FILE simple_pore.py MAX_NUM_PROC 1)
-python_test(FILE field_test.py MAX_NUM_PROC 1)
-python_test(FILE lb_boundary.py MAX_NUM_PROC 2)
-python_test(FILE lb_streaming.py MAX_NUM_PROC 4)
-
-if(PY_H5PY)
-  python_test(FILE h5md.py MAX_NUM_PROC 2)
-endif(PY_H5PY)
-
-add_custom_target(python_test_data
-  COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/data ${CMAKE_CURRENT_BINARY_DIR}/data
-  COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/tests_common.py ${CMAKE_CURRENT_BINARY_DIR}
-  COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/virtual_sites_tracers_common.py ${CMAKE_CURRENT_BINARY_DIR}
-  COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/ek_common.py ${CMAKE_CURRENT_BINARY_DIR}
-  COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/ek_eof_one_species_base.py ${CMAKE_CURRENT_BINARY_DIR}
-)
-
-add_custom_target(check_python_serial COMMAND ${CMAKE_CTEST_COMMAND} $(ARGS) -C serial --output-on-failure)
-add_dependencies(check_python_serial pypresso python_test_data)
-
-add_custom_target(check_python_parallel COMMAND ${CMAKE_CTEST_COMMAND} $(ARGS) -C parallel --output-on-failure)
-add_dependencies(check_python_parallel pypresso python_test_data)
-
-add_custom_target(check_python_parallel_odd COMMAND ${CMAKE_CTEST_COMMAND} $(ARGS) -C parallel_odd --output-on-failure)
-add_dependencies(check_python_parallel_odd pypresso python_test_data)
-
-add_custom_target(check_python)
-add_dependencies(check_python pypresso python_test_data check_python_serial)
-
-add_dependencies(check check_python)
+add_subdirectory(python)
```

Comment:
we lost git history here... better: ```git mv CMakeLists.txt python```

Reply:
I see no difference between `testsuite/CMakeLists.txt` in upstream and `testsuite/python/CMakeLists.txt` in this PR. Commit c273d64 shows everything was renamed with `git mv`. Commit [8e80916](https://github.com/espressomd/espresso/pull/2268/commits/8e80916691bf59fa41c566a57180d339a3a86a1c) applied the changes by junghans to preserve git history.